File: CSharp part2/03. Methods/07.ReverseNumber/ReverseNumber.cs
// Write a method that reverses the digits of given decimal number.

using System;

class ReverseNumber
{
    static string ReverseDigits(string number)
    {
        char[] numberReversal = new char[number.Length];
        string reversedNumber = "";
        foreach (char element in number)
        {
            reversedNumber = element + reversedNumber;
        }
        return reversedNumber;
    }
    static void Main()
    {
        Console.Write("Enter a number:");
        string number = Console.ReadLine();
        Console.WriteLine("Digits reversed:{0}", ReverseDigits(number));
    }
}

File: JavaScript-Fundamentals/Functions/02.ReverseNumber/ReverseNumber.js
// Write a function that returns the last digit of given integer as an English word.

var jsConsole;
var btn = document.getElementById('reverseNumber');
btn.onclick = reverseNumber;

function reverseNumber () {
	var input = jsConsole.readInteger('#n');
	var inputAsString = input.toString();
	var reversedNumber = [inputAsString.length];
	for (var i = 0, k = inputAsString.length-1; k>=0 ; i++,k--) {
		reversedNumber[i] = inputAsString [k];
	}
	
	jsConsole.writeLine("Reversed number is "+reversedNumber);
}

